proto/cosmos/bank/v1beta1/genesis.proto

```diff
@@ -22,6 +22,9 @@ message GenesisState {
 
   // denom_metadata defines the metadata of the differents coins.
   repeated Metadata denom_metadata = 4 [(gogoproto.moretags) = "yaml:\"denom_metadata\"", (gogoproto.nullable) = false];
+
+  // blocked_addrs is an array containing the blocked addresses
+  repeated string blocked_addrs = 5 [(gogoproto.moretags) = "yaml:\"blocked_addrs\"", (gogoproto.nullable) = false];
```

Comment:
I suggest another name: `blocked_addresses`, since using full `Address` instead of abbreviated `Addr` seems the sdk's convention.